goonstation.dme

```diff
@@ -1427,7 +1429,6 @@ var/datum/preMapLoad/preMapLoad = new
 #include "code\obj\item\plank.dm"
 #include "code\obj\item\plants.dm"
 #include "code\obj\item\playing_cards.dm"
-#include "code\obj\item\mono_card_game.dm"
```

Comment:
This isn't much of a problem as it is funny; apparently this got into mainline gooncode? lol

Reply:
Yep, I left it in because well, it is correct.